modules/router-store/src/router_store_module.ts

```diff
@@ -155,6 +156,16 @@ export class StoreRouterConnectingModule {
   >(
     config: StoreRouterConfig<T> = {}
   ): ModuleWithProviders<StoreRouterConnectingModule> {
+    if (!isNgrxMockEnvironment() && isDevMode()) {
```

Comment:
At the moment this warning will be logged everytime, right?
I think we only want it when the `DefaultRouterStateSerializer` is used, and preferably together when there are some enabled `runtimeChecks`.

Reply:
I'm now injecting the `ACTIVE_RUNTIME_CHECKS` from ngrx/store to show the warning only when the conflicting `runtimeChecks` are enabled